Packs/RSANetWitness_v11_1/IncidentFields/incidentfield-Metas_Events.json

```diff
@@ -0,0 +1,43 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "metasevents",
+    "closeForm": false,
+    "columns": [
+        {
+            "displayName": "Column Header 1",
+            "fieldCalcScript": "",
+            "isDefault": true,
+            "isReadOnly": false,
+            "key": "columnheader1",
+            "orgType": "shortText",
+            "required": false,
+            "script": "",
+            "selectValues": null,
+            "type": "shortText",
+            "width": 150
+        }
+    ],
+    "content": true,
+    "defaultRows": [],
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_metasevents",
+    "isReadOnly": true,
+    "locked": false,
+    "name": "Metas Events",
+    "neverSetAsRequired": false,
+    "openEnded": false,
+    "ownerOnly": false,
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "grid",
+    "unmapped": false,
+    "unsearchable": false,
```

Comment:
Yes so please update it to true. I know that it is a bit confusing because the name of the field is **un**searchable and since we do not want the user the search it should be true.

Reply:
but I want them to be able to search